schema/me/conversation.js

```diff
@@ -37,25 +37,54 @@ export const BuyerOutcomeTypes = new GraphQLEnumType({
   },
 })
 
+export const AttachmentType = new GraphQLObjectType({
+  name: "AttachmentType",
+  desciption: "Fields of an attachment (currently from Radiation)",
+  fields: {
+    id: {
+      description: "Attachment id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    content_type: {
+      description: "Content type of file.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    file_name: {
+      description: "File name.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    download_url: {
+      descrpition: "URL of attachment.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+})
+
 export const MessageType = new GraphQLObjectType({
   name: "MessageType",
   description: "A message in a conversation.",
   fields: {
     id: {
-      description: "Impulse id.",
+      description: "Impulse message id.",
       type: new GraphQLNonNull(GraphQLString),
     },
-    from_email_address: {
-      description: "Email address of sender.",
+    radiation_message_id: {
       type: new GraphQLNonNull(GraphQLString),
     },
-    snippet: {
-      description: "A snippet of the full message",
-      type: new GraphQLNonNull(GraphQLString),
+    is_from_user: {
+      description: "True if message is from the user to the partner.",
+      type: GraphQLBoolean,
+      resolve: ({ from_email_address, conversation_from_address }) => {
+        from_email_address === conversation_from_address
+      },
     },
-    radiation_message_id: {
+    raw_text: {
```

Comment:
This replaces the `snippet` and is where we'll be doing sanitization, presumably.